src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -73,6 +82,76 @@ public static void RegisterAsScrollPort(UIElement element)
 
 		private void Initialize()
 		{
+			SubscribeToOverridenRoutedEvents();
+		}
+
+		private void SubscribeToOverridenRoutedEvents()
+		{
+			// Overridden Events are registered from constructor to ensure they are
+			// registered first in event handlers.
+			// https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.onpointerpressed#remarks
+
+			var implementedEvents = GetImplementedRoutedEvents();
+
+			if (implementedEvents.HasFlag(RoutedEventFlag.BringIntoViewRequested))
+			{
+				BringIntoViewRequested += OnBringIntoViewRequestedHandler;
+			}
+		}
+
+		protected RoutedEventFlag GetImplementedRoutedEvents()
+		{
+			// TODO: GetImplementedRoutedEvents() should be evaluated at compile-time
+			// and the result placed in a partial file.
+			var type = GetType();
+
+			if (ImplementedRoutedEvents.TryGetValue(type, out var result))
+			{
+				return result;
+			}
+
+			RoutedEventFlag implementedRoutedEvents;
+
+			var baseClass = type.BaseType;
+			if (baseClass == null || type == typeof(Control) || type == typeof(UIElement))
```

Comment:
I don't get this condition? If we inherit directly from `Control` or `UIElement` I think that we should still call `EvaluateImplementedRoutedEvents`

Reply:
This checks if the baseClass is null (that should never happen), or the type itself is `Control` or `UIElement`, so that should be safe - these have only empty virtual methods

Reply:
(miss-read, the second part of the checks are on the `type` itself ... then it makes sense!)